src/index.html

```diff
@@ -8,15 +8,28 @@
     <link rel="stylesheet" href="resetCSS.css">
     <link href="https://fonts.googleapis.com/css?family=Roboto:400,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
+    <script src="../docs/js/bundle.js" async></script>
 </head>
 <body>
     <main>
         <h1>The most awesome Weather App ever created</h1>
         <section class="section inputBox">
-            <!-- input section -->
+            <form class="inputs">
+                <select type="text" class="input shortInput" id="js_countryCode"></select>
+                <select type="text" class="input" id="js_optSelect">
+                    <option selected>City name</option>
+                    <option>Geo coordinates</option>
+                    <option>ZIP Code</option>
+                </select>
```

Comment:
`<select>` nie potrzebuje `type`, ale przydałoby się, żeby miał name. Z kolei każda z `<option>` powinna mieć `value`. Z `value` będzie znacznie łatwiej z nimi pracować, manipulując DOM-em. Najlepiej by było, jeśli `value` odpowiadałyby opcjom ze switcha w funkcji Łukasza.

Reply:
Powinniśmy też trzymać wszystko to, co będziemy pokazywać w folderze docs, a tylko JS-ową logikę w src.